linux/scripts/launch.sh

```diff
@@ -21,14 +21,20 @@
 # SOFTWARE.
 
 
-# This script is run in a Linux guest. If you are using s2e-env, it will be
-# automatically run at user login.
+# This script is run in a Linux guest. If you built your image with s2e-env it
+# will automatically run at user login.
 #
-# If the image is run in non-S2E mode a snapshot will infinitely loop at
-# s2eget. This gives the user the opportunity to take a snapshot so that when
-# the image is rebooted into S2E mode it will retrieve the bootstrap script and
+# If the image is run in non-S2E mode execution will loop indefinitely on
+# s2eget. This gives the user the opportunity to take a snapshot. When the
+# image is rebooted into S2E mode it will retrieve the bootstrap script and
 # start executing it immediately.
 
+echo "booted kernel $(uname -r)" > /dev/ttyS0
+
+# Receive a command from the host. Execution continues regardless of whether a
+# command is received or not
+read -t 5 cmd < /dev/ttyS0
```

Comment:
1. If the host is too slow then the read command will time out. The launch script will continue as normal because I wanted to ensure that the guest's overall behaviour remains unchanged (i.e. default behaviour when running an analysis is that no commands will be sent from host to guest). It is up to the host (in my case `s2e-env`) to handle this error.

2. cmd is never used. I didn't realise you could call `read` with no variables. I have updated the PR.